packages/flutter/test/material/text_field_test.dart

```diff
@@ -250,6 +250,75 @@ void main() {
     await checkCursorToggle();
   });
 
+  testWidgets('cursor has expected defaults', (WidgetTester tester) async {
+    await tester.pumpWidget(
+        overlay(
+          child: const TextField(
+            cursorRadius: Radius.circular(3.0),
+          ),
+        )
+    );
+
+    final TextField textField = tester.firstWidget(find.byType(TextField));
+    expect(textField.cursorWidth, 2.0);
+    expect(textField.cursorRadius, const Radius.circular(3.0));
+  });
+
+  testWidgets('cursor layout has correct width', (WidgetTester tester) async {
+    final TextEditingController controller = new TextEditingController();
+
+    await tester.pumpWidget(
+        overlay(
+          child: new RepaintBoundary(
+            key: const ValueKey<int>(1),
+            child: new TextField(
+              controller: controller,
+              cursorWidth: 15.0,
+            ),
+          ),
+        )
+    );
+    expect(controller.selection.baseOffset, -1);
```

Comment:
What do these controller.selection values have to do with checking the cursor's width?

Reply:
I copied these from another test, but you're right that they don't need to be there. I removed them.